### DISCUSSION TITLE
Raspberry Pi Just released an official AI HAT, 13 TOPS

### DISCUSSION THREAD

**Discussion by dpicassom** [geezacoleman/OpenWeedLocator#132] [General]
Raspberry Pi Just released an official AI HAT, 13 TOPS:

[Raspberry Pi AI Kit available now at $70](https://www.raspberrypi.com/news/raspberry-pi-ai-kit-available-now-at-70/)

Much more processing power.

**Comment by geezacoleman** [comment]
Yes, this is great news! Will certainly be trying it out for the OWL. The biggest challenges seem to be in Hailo model type and converting existing models to that format (successfully).

It will certainly speed up inference for in-crop detection!
